projects/batfish/src/main/java/org/batfish/main/Batfish.java

```diff
@@ -841,8 +841,7 @@ private void checkQuestionsDirExists() {
    }
 
    @Override
```

Comment:
You should not claim to throw more than you do throw, and yes, it's compatible to throw fewer exceptions than the method you're overriding.

(In this particular case, the compile warning was that the method might throw InterruptedException, which is not semantically legal in a close method).